Reenable assertions (#1609)
klee/klee: CMakeLists.txt

option(ENABLE_KLEE_ASSERTS "Enable KLEE assertions" ON)
if (ENABLE_KLEE_ASSERTS)
  message(STATUS "KLEE assertions enabled")

>   # We have to add the undefine to the flags, otherwise "-D-UNDEBUG" will be added

Reviewer: ```suggestion
  # We have to undefine the `NDEBUG`flag, which is the default added by CMake
```

Author: I was also at first puzzled by this comment, but it refers to why `-UNDEBUG` cannot be added to `KLEE_COMPONENT_CXX_DEFINES`, just like `-DNDEBUG` below.

Also, I think `NDEBUG` is defined by the compiler, but I will have to check again.
